### DISCUSSION TITLE
Restructuring of the project files

### DISCUSSION THREAD

**Discussion by okhuaroboosayuki** [NaijaCodePadi/ShareAm#26] [General]
Hello sir @Amospikins 

Kindly permit @Ericademola and I to restructure the whole project files again for effective understanding for everyone

**Comment by Amospikins** [comment]
How do I do that ... What's the procedure

**Reply by okhuaroboosayuki** [reply]
I'm talking about restructuring the folders and CSS, so that everybody works on a separate CSS file:

like auth CSS will be in the auth folder, dashboard and so on. And structuring folders like images, fonts
